installer/dbscript/dbscript-2.sql

```diff
@@ -4610,3 +4610,9 @@ SET @auth_admin_verify_data_group_id := (SELECT `id` FROM ohrm_data_group WHERE
 
 INSERT INTO ohrm_user_role_data_group (`can_read`, `can_create`, `can_update`, `can_delete`, `self`, `data_group_id`,`user_role_id`)
 VALUES (1, 0, 0, 0, 0, @auth_admin_verify_data_group_id, @admin_role_id);
+
+ALTER TABLE `ohrm_timesheet_action_log` CHANGE `performed_by` `performed_by` INT(20) NULL DEFAULT NULL;
+
+ALTER TABLE `ohrm_timesheet_action_log` DROP FOREIGN KEY `ohrm_timesheet_action_log_ibfk_1`;
+
+ALTER TABLE `ohrm_timesheet_action_log` ADD CONSTRAINT `ohrm_timesheet_action_log_performed_by_id` FOREIGN KEY (`performed_by`) REFERENCES `ohrm_user`(`id`) ON DELETE SET NULL ON UPDATE RESTRICT;
```

Comment:
should we include `ON UPDATE RESTRICT` @RajithaKumara ? we got this query from phpmyadmin when changing the table

Reply:
Better to check DB compatibility with suported min max versions. e.g. MySQL 5.5 & MySQL 5.7, MariaDB 5.5 & MariaDB 10.6

Reply:
MySQL 5.5 have support http://download.nust.na/pub6/mysql/doc/refman/5.5/en/innodb-foreign-key-constraints.html
MySQL 5.7 have https://dev.mysql.com/doc/refman/5.7/en/create-table-foreign-keys.html
MariaDB 5.5 have https://mariadb.com/kb/en/constraint/
MariaDB 10.6 have https://mariadb.com/kb/en/foreign-keys/